Sources/Card/Models/Card.swift

```diff
@@ -1,32 +1,60 @@
 import Foundation
 
-/// Initialize a card object
-/// - Parameters:
-///   - number: The full card number, or PAN.
-///   - expirationMonth: The card's expiration month in MM format.
-///   - expirationYear: The card's expiration year in YYYY format.
-///   - securityCode: The card's security code (CVV, CVC, CVN, CVE, or CID).
 public struct Card: Encodable {
 
-    public var number: String
-    public var expirationMonth: String
-    public var expirationYear: String
-    public var securityCode: String
-
     enum CodingKeys: String, CodingKey {
         case number
         case expiry
         case securityCode
+        case cardHolderName = "name"
+        case billingAddress
     }
 
+    /// The primary account number (PAN) for the payment card.
+    public var number: String
+
+    /// The card expiration month
+    public var expirationMonth: String
+
+    /// The card expiration year
+    public var expirationYear: String
+
+    /// The three- or four-digit security code of the card. Also known as the CVV, CVC, CVN, CVE, or CID.
+    public var securityCode: String
+
+    /// The card holder's name as it appears on the card.
+    public var cardHolderName: String?
+
+    /// The portable international postal address. Maps to [AddressValidationMetadata](https://github.com/googlei18n/libaddressinput/wiki/AddressValidationMetadata) and HTML 5.1 [Autofilling form controls: the autocomplete attribute](https://www.w3.org/TR/html51/sec-forms.html#autofilling-form-controls-the-autocomplete-attribute).
```

Comment:
Same comment as above about the links to validation and autofilling. I don't know that these are needed.